src/lib.rs

```diff
@@ -1,10 +1,25 @@
-use std::char;
+use std::{char, cmp};
+
+const MIN_LONGITUDE: f64 = -180.;
+const MAX_LONGITUDE: f64 = 180.;
+const MIN_LATITUDE: f64 = -90.0;
```

Comment:
You can leave off the trailing `0`, but I like to be consistent. Will push a fix shortly.